doc/sphinx/proof-engine/ltac.rst

```diff
@@ -268,100 +246,124 @@ Goal selectors
 
 .. todo: mention this applies to Print commands and the Info command
 
+Normally tactic expressions are applied only to the current goal.  Goal
+selectors provide a way to apply a tactic expression to another goal or multiple
+goals.
```

Comment:
This looks good but would benefit from additionally referring to the `Default Goal Selector` option.